package.json

```diff
@@ -33,70 +33,70 @@
-    "@babel/preset-env": "^7.5.4",
+    "@babel/preset-env": "^7.6.0",
     "@babel/preset-flow": "^7.0.0",
     "@babel/preset-react": "^7.0.0",
-    "@babel/runtime": "^7.5.4",
+    "@babel/runtime": "^7.6.0",
     "@crave/farmblocks-dev-scaffold": "^1.2.0",
     "@percy-io/percy-storybook": "^2.1.0",
-    "@storybook/addon-actions": "^5.1.9",
-    "@storybook/addon-info": "^5.1.9",
-    "@storybook/addon-links": "^5.1.9",
-    "@storybook/addon-storyshots": "^5.1.9",
-    "@storybook/addons": "^5.1.9",
-    "@storybook/react": "^5.1.9",
-    "@storybook/theming": "^5.1.9",
-    "@testing-library/jest-dom": "^4.0.0",
-    "@testing-library/react": "^8.0.4",
-    "@testing-library/react-hooks": "^1.1.0",
+    "@storybook/addon-actions": "^5.1.11",
+    "@storybook/addon-info": "^5.1.11",
+    "@storybook/addon-links": "^5.1.11",
+    "@storybook/addon-storyshots": "^5.1.11",
+    "@storybook/addons": "^5.1.11",
+    "@storybook/react": "^5.1.11",
+    "@storybook/theming": "^5.1.11",
+    "@testing-library/jest-dom": "^4.1.0",
+    "@testing-library/react": "^9.1.4",
+    "@testing-library/react-hooks": "^2.0.1",
     "babel-core": "^7.0.0-bridge.0",
-    "babel-eslint": "^10.0.2",
-    "babel-jest": "^24.8.0",
+    "babel-eslint": "^10.0.3",
+    "babel-jest": "^24.9.0",
     "babel-loader": "^8.0.6",
     "babel-plugin-require-context-hook": "^1.0.0",
     "babel-plugin-styled-components": "^1.10.6",
-    "commitizen": "^4.0.1",
-    "coveralls": "^3.0.4",
+    "commitizen": "^4.0.3",
+    "coveralls": "^3.0.6",
     "cz-lerna-changelog": "^2.0.2",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "eslint": "^6.0.1",
+    "eslint": "^6.3.0",
     "eslint-config-airbnb": "^17.1.1",
-    "eslint-config-prettier": "^6.0.0",
+    "eslint-config-prettier": "^6.2.0",
     "eslint-config-standard": "^13.0.1",
     "eslint-plugin-babel": "^5.3.0",
-    "eslint-plugin-import": "^2.18.0",
-    "eslint-plugin-jest": "^22.7.2",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-jest": "^22.17.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
     "eslint-plugin-node": "^9.1.0",
     "eslint-plugin-prettier": "^3.1.0",
     "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-react": "^7.14.2",
-    "eslint-plugin-standard": "^4.0.0",
-    "jest": "^24.8.0",
-    "jest-enzyme": "^7.0.2",
+    "eslint-plugin-react": "^7.14.3",
+    "eslint-plugin-standard": "^4.0.1",
+    "jest": "^24.9.0",
+    "jest-enzyme": "^7.1.1",
     "jest-styled-components": "^6.3.3",
-    "lerna": "3.16.0",
+    "lerna": "3.16.4",
     "prettier": "^1.18.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.9.0",
+    "react-dom": "^16.9.0",
     "react-spring": "^8.0.27",
-    "react-test-renderer": "^16.8.6",
+    "react-test-renderer": "^16.9.0",
     "recompose": "^0.30.0",
-    "snyk": "^1.192.4",
+    "snyk": "^1.226.1",
     "styled-components": "^4.3.2"
   },
   "resolutions": {
```

Comment:
We can remove this. It is here because some dependencies used versions of these packages that had some kind of vulnerability.
I checked with `yarn why` and the dependencies already use the fixed versions.